src/components/channels_list.js

```diff
@@ -40,8 +40,7 @@ class ChannelsList extends Component {
     componentWillReceiveProps(props) {
         if (props.channels.current_channel_id &&
           !this.props.channels.current_channel_id) {
-            console.warn('TODO route to PostsList'); // eslint-disable-line no-console
-            // TODO: Routes.PostLists(); // eslint-disable-line new-cap
+            Routes.PostsList(); // eslint-disable-line new-cap
```

Comment:
I would rather create a wrapper around the router/routes/route-actions so we can call `Router.go('PostsList');`. By naming our routes like functions I think we'll end up with confusing name overloading.
e.g. `Actions.selectTeam();` and `props.actions.selectTeam();`.